<launch>
    <node name="gpio_out" pkg="py_control" type="gpio_out.py" output="screen"/>
    <node name="gpio_in" pkg="py_control" type="gpio_in.py" output="screen"/>
</launch>
